akvo/iati/exports/elements/conditions.py

```diff
@@ -22,9 +22,10 @@ def conditions(project):
         conditions_element.attrib['attached'] = '0'
 
     for condition in project.conditions.all():
-        if condition.type:
-            condition_element = etree.SubElement(conditions_element, "condition")
-            condition_element.attrib['type'] = condition.type
+        if condition.type or condition.text:
```

Comment:
This leads to a possible error on line 34 if condition.type is False, since condition_element then won't be assigned to.